mino/proxy/http/mod.go

```diff
@@ -97,6 +105,11 @@ func (h HTTP) Stop() {
 	h.quit <- struct{}{}
 }
 
+// GetAddr implements proxy.Proxy.
+func (h HTTP) GetAddr() net.Addr {
+	return h.ln.Addr()
```

Comment:
calling this before `Listen()` could result in `h.ln` not being initialized.
Should we guard against that case ?